### PR TITLE
fix json literals in abi

### PR DESCRIPTION
**Pull request by BitBaseBit**

### CODE REVIEW

**Comment by murcake** [comment]
Does that need to go to `main`?

**Review by swapnilraj — APPROVED** [review]
Looks good, can you just a small note about this change in the docstring

File: warp/yul/utils.py [hunk 1]
```diff
@@ -197,4 +197,6 @@ def make_abi_StarkNet_encodable(abi):
     """
     abiStr = f"{abi}".replace("'address'", "'uint256'")
     abiStr = abiStr.replace("'", '"')
+    abiStr = abiStr.replace("True", "true")
+    abiStr = abiStr.replace("False", "false")
```
Comment (murcake) [inline warp/yul/utils.py:201]: Why do we do that? Also I'm worried about it changing function names like `isTrue` to `istrue`, which would change the selector again

Comment (BitBaseBit) [reply warp/yul/utils.py:201]: oh yeah, I'll fix it now

**Pull request closed without merging**